.eslintignore

```diff
@@ -1 +1,3 @@
 coverage
+node_modules
+support
```

Comment:
Better question would be do you see anything other than `describe-operation-hooks.js` under `support`?